src/SFML/Graphics/Text.cpp

```diff
@@ -65,22 +65,19 @@ void addGlyphQuad(sf::VertexArray& vertices, sf::Vector2f position, const sf::Co
 {
     const float padding = 1.0;
 
-    const float left   = glyph.bounds.left - padding;
-    const float top    = glyph.bounds.top - padding;
-    const float right  = glyph.bounds.left + glyph.bounds.width + padding;
-    const float bottom = glyph.bounds.top + glyph.bounds.height + padding;
-
-    const float u1 = static_cast<float>(glyph.textureRect.left) - padding;
-    const float v1 = static_cast<float>(glyph.textureRect.top) - padding;
-    const float u2 = static_cast<float>(glyph.textureRect.left + glyph.textureRect.width) + padding;
-    const float v2 = static_cast<float>(glyph.textureRect.top + glyph.textureRect.height) + padding;
-
-    vertices.append({{position.x + left - italicShear * top, position.y + top}, color, {u1, v1}});
-    vertices.append({{position.x + right - italicShear * top, position.y + top}, color, {u2, v1}});
-    vertices.append({{position.x + left - italicShear * bottom, position.y + bottom}, color, {u1, v2}});
-    vertices.append({{position.x + left - italicShear * bottom, position.y + bottom}, color, {u1, v2}});
-    vertices.append({{position.x + right - italicShear * top, position.y + top}, color, {u2, v1}});
-    vertices.append({{position.x + right - italicShear * bottom, position.y + bottom}, color, {u2, v2}});
+    const sf::Vector2f p1 = glyph.bounds.position - sf::Vector2f(padding, padding);
```

Comment:
`sf::Vector2f(padding, padding)` is used everywhere, maybe change the above `float` to a `Vector2f`?